warehouse/warehouse.go

```diff
@@ -640,6 +693,7 @@ func (wh *HandleT) mainLoop() {
 		}
 
 		warehouses := wh.warehouses
+		wh.sortWarehousesByLastEventAt(warehouses)
```

Comment:
Instead of sorting it here, let's use this logic to create upload jobs in the uploadJobAllocator?